--- sudoers	2021-04-03 03:40:20.202062250 -0700
+++ ../sudoers	2021-04-03 02:45:47.815379837 -0700
@@ -84,6 +84,9 @@
 ## Same thing without a password
 # %wheel ALL=(ALL) NOPASSWD: ALL
 
+## Run these commands using sudo and without a password
+kevin ALL=(ALL) NOPASSWD:/bin/shutdown,/bin/pacman,/bin/vim
+
 ## Uncomment to allow members of group sudo to execute any command
 # %sudo	ALL=(ALL) ALL
 
